hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -103,58 +107,62 @@ def lambda_handler(event, context):
                 )
                 privacy_group_list = metadata.get("privacy_groups", [])
                 metadata["privacy_groups"] = filter(
-                    lambda x: get_privacy_group_matcher_active(str(x)),
+                    lambda x: get_privacy_group_matcher_active(
+                        str(x), time.time() // CACHED_TIME
```

Comment:
If you are using time.time(), wouldn't that keep changing the argument list? How is this caching? Seems like every call is cache-busted. I'm not following.

Reply:
I am actually using ```time.time() // CACHED_TIME```, ```CACHED_TIME``` is default to 300. So ```time.time() // CACHED_TIME``` will change every 5 mins(300 seconds), which means after 5 mins, the cached result will be expired

Reply:
Added a comment to explain why using ```time.time() // CACHED_TIME```, will consider to optimize in the v1 release